test/specs/payload.specs.ts

```diff
@@ -58,7 +57,7 @@ describe("Payload Helpers", () => {
                 {
                     foo: "bar",
                     foo2: {
-                        bar: new Buffer(["test"]),
+                        bar: Buffer.from("test"),
```

Comment:
`new Buffer` is deprecated. Not an important change though. It should not impact the tests.